test/helpers/page_context_helper_test.rb

```diff
@@ -64,6 +64,8 @@ class PageContextHelperTest < ActionView::TestCase
   end
 
   it 'should return project page context' do
+    @account = create(:admin)
+    stubs(:current_user).returns(@account)
```

Comment:
This is the same as writing : `current_user = @account`. If you remove this line, this test should behave the same. Try running the test without these lines.

Reply:
stubs(:current_user).returns(@account) removing this line causing test case failure. So , I am not making this change.

Reply:
What is the error? Always mention the actual error instead of stating that something causes an error.

Reply:
If there is no error and this is simply fixing the test, then we should know how this fixes the test. 
Add a byebug in the controller and check if `current_user` in the controller is the same as `@account` from the test. This would mean that this stubs line actually is stubbing inside the controller.